recipes/rapids/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Please delete this manually packaged license file. It's included in the source distribution from pypi

Reply:
Removed license file as suggested